CHANGELOG.md

```diff
@@ -9,9 +9,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- [#691](https://github.com/nf-core/sarek/pull/691) - Enable `PROFILE=conda`, `PROFILE=docker` and `PROFILE=singularity` for pytest
+
 ### Changed
 
 - [#679](https://github.com/nf-core/sarek/pull/679) - Back to `dev`
+- [#691](https://github.com/nf-core/sarek/pull/691) - To run the same pytest as before locally, use `PROFILE=docker`
```

Comment:
```suggestion
- [#691](https://github.com/nf-core/sarek/pull/691) - To run the same pytest as before locally, use TMPDIR=~/tmp, and `PROFILE=conda`, `PROFILE=docker` or `PROFILE=singularity`. 
```
or something like that.

Reply:
Before we had just `docker`, all the rest is new, let's not document that too much for now, as we're still figuring it out ;-)